skimage/transform/_warps.py

```diff
@@ -16,6 +16,54 @@
 )
 
 
+def _preprocess_resize_output_shape(image, output_shape):
+    """Validate resize output shape according to input image.
+
+    Parameters
+    ----------
+    image: ndarray
+        Image to be resized.
+    output_shape: tuple or ndarray
+        Size of the generated output image `(rows, cols[, ...][, dim])`. If
+        `dim` is not provided, the number of channels is preserved.
+
+    Returns
+    -------
+    image: ndarray
+        The input image reshaped if its number of dimensions is not
+        equal to output_shape_length.
+    output_shape: tuple
+        The output image converted to tuple.
+
+    Raises
+    ------
+    ValueError:
+        If output_shape length is smaller than the image number of
+        dimensions
+
+    Notes
+    -----
+    The input image is reshaped if its number of dimensions is not
+    equal to output_shape_length.
+
+    """
+    output_shape = tuple(np.ravel(output_shape))
```

Comment:
What usecase are you trying to ensure works here? why not omit this line?

Reply:
i guess, if you want an `ndarray`, then `tuple(output_shape)` is good enough right?

Reply:
It allows providing output_shape as an ndarray with shape (1, n)
```python
>>> tuple(np.ones((1, 3)))
(array([1., 1., 1.]),)
>>> tuple(np.ravel(ones((1, 3))))
(1.0, 1.0, 1.0)
```

Reply:
> It allows providing output_shape as an ndarray with shape (1, n)

And this is desirable because...? 😂 

I'll note that the conversation over at the Array object data API suggests that `.shape` *must* be a 1D object:

https://data-apis.github.io/array-api/latest/API_specification/array_object.html#shape

I suggest we stick to that.

Reply:
Yes, I agree with you and :-1: with my old me :stuck_out_tongue_closed_eyes:. I will modify this ASAP

Reply:
It's fixed! Thank you @hmaarrfk and @jni ;)